optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/employee/EmployeeController.java

```diff
@@ -84,6 +98,40 @@ public EmployeeController(EmployeeService employeeService) {
         return new ResponseEntity<>(employeeService.createEmployee(tenantId, employeeView), HttpStatus.OK);
     }
 
+    @ApiOperation("Import employees from an Excel file")
+    @PostMapping("/import")
+    public ResponseEntity<List<Employee>> addEmployeesFromExcelFile(@PathVariable @Min(0) Integer tenantId,
+                                                                    @RequestParam("file") MultipartFile excelDataFile)
+            throws IOException {
+
+        List<EmployeeView> excelEmployeeList = employeeListXlsxFileIO
```

Comment:
All of this belongs to `EmployeeService`. Motivation:
- Less dependencies for the controller (only the service).
- Clear separation of concerns. Controller = network layer, Service = application layer.
- Imagine we have a CLI in addition to REST API. The CLI shouldn't depend on a REST controller in order to read employees from a file. The CLI should only rely on the service to do that.